File: Prototyper/EpicAlgo420Framework/src/Data/Model/Actor.java
package Data.Model;

public class Actor{

    private String fornavn, etternavn;
    private int alder;
    
        public Actor(String fornavn, String etternavn, int alder) {
            this.fornavn = fornavn;
            this.etternavn = etternavn;
            this.alder = alder;
        }
    
        public String getFornavn() {
            return fornavn;
        }
    
        public void setFornavn(String fornavn) {
            this.fornavn = fornavn;
        }
    
        public String getEtternavn() {
            return etternavn;
        }
    
        public void setEtternavn(String etternavn) {
            this.etternavn = etternavn;
        }
    
        public int getAlder() {
            return alder;
        }
    
        public void setAlder(int alder) {
            this.alder = alder;
        }
    
        @Override
        public String toString() {
    
           return getFornavn() + " " + getEtternavn() + " - " + " " + "Alder" + ": " + getAlder();
    
        }


    
    
    
    }
    

File: EpicAlgo/EpicAlgo/EpicAlgo/Models/Actor.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace EpicAlgo.Models
{
    public class Actor
    {

        public String Fornavn { get; set; }
        public String Etternavn { get; set; }
        public int Alder { get; set; }

    public Actor(String fornavn, String etternavn, int alder)
        {
            this.Fornavn = fornavn;
            this.Etternavn = etternavn;
            this.Alder = alder;
        }


        public override string ToString()
        {
            return Fornavn + " - " + Etternavn + " - " + Alder;
        }

    }
}
